oauth2_http/java/com/google/auth/oauth2/S2A.java

```diff
@@ -0,0 +1,112 @@
+package com.google.auth.oauth2;
+
+import com.google.api.client.http.GenericUrl;
+import com.google.api.client.http.HttpRequest;
+import com.google.api.client.http.HttpResponse;
+import com.google.api.client.json.JsonObjectParser;
+import com.google.api.client.util.GenericData;
+import com.google.auth.http.HttpTransportFactory;
+import com.google.common.collect.Iterables;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.ServiceLoader;
+import javax.annotation.concurrent.ThreadSafe;
+
+/**
+ * Utilities to fetch the S2A (Secure Session Agent) address from the mTLS configuration.
+ *
+ * <p>mTLS configuration is queried from the MDS MTLS Autoconfiguration endpoint.
+ */
+@ThreadSafe
+public final class S2A {
+  public static final String MTLS_CONFIG_ENDPOINT =
```

Comment:
```suggestion
  public static final String S2A_CONFIG_ENDPOINT_POSTFIX =
```

Reply:
Done in https://github.com/googleapis/google-auth-library-java/pull/1400/commits/bce602e84717d76c284e55e3cd6c8c69c99007e9